### ISSUE TITLE
Facilities - Sprint 59

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9183]
<details><summary>Background</summary>

🥅   — Sprint goals (pending Sprint planning)
💡 — Design
💧  — Drupal engineering
🎉 — FE
🎓  — Training / Change management / KB
📟 — Helpdesk
🔮  —  Planning / roadmap / epic refinement
📊  —  Analytics
🚰  — Service design
🌱 — Good first task

</details>

## #9096 Facility COVID status post-launch

Show COVID status in FL
- #9033 🥅
- #9161 🥅

CMS 
- #9160
- #9099
- #9168 (Ethan)

Change management
- [ ] Candidate #9195 
- [ ] Others TBD
- [ ] Track updates that may be required for VAMC training

VA Design system updates (stretch)
- Consistent padding / spacing
- Text and carat wrapping issues

## Facility Operating status

- #6319 (Design flow in 59, implement in 60)

## Facility Locator

<details><summary>Lower priority candidates for Q2</summary>

- [ ] #8769 (Val/Kev)
- [ ] #8870 (still blocked by Lighthouse)
- [ ] Bounding box to Radius got rolled back to bounding box, possibly low priority to go back to radius.
- [ ] Get to know Provider Profile Management System - Data source for Community care providers, Urgent Care, Community pharmacies. VAOS (VA Online Scheduling) also consumues this endpoint in Vets API

</details>

## #7622 Lovell

Background: See epic https://github.com/department-of-veterans-affairs/va.gov-team/issues/34022

### Research

Ready for PO review
- [x] #8895 
- [x] #8995

Next steps
- [x] #9065 
- [x] Then #9069
- [ ] Then #9070

<details><summary>CMS (on pause until design is validated)</summary>

- [x] #7626 (sprint 55)
- [ ] Create proof of concept
- [ ] CMS design discovery epic
  - [ ] #8863
- [ ] Refine epic #1939 (for DoD facilities)
  - [ ] #1940 
  - [ ] #1941 
  - [ ] #1942 
  - [ ] #1943  

</details> 

## Hours

[Mural for reference](https://app.mural.co/t/vagov6717/m/vagov6717/1653327582500/26b235b5a21b63f6f30cf209d51303f53062c0d4?sender=kevinwalsh8610)

### #6931 Deprecate table-based hours in favor of office hours field

- [ ] #8861 PR ready, but waiting for 8935
- [ ] #8935 (Jay)
- [ ] Design short and medium-term solutions for empty day values, and FE Defensiveness
  - [ ] Next steps when Saturday / Sunday is empty - Change management? Design/ Audit?
  - [ ] Should all rows be required?  

### #4888 Improve hours UX

Discuss if action is needed:
- [ ] #8474

### Hours component

needs refining. 

###  #3151 3 VAMC Top task pages

Billing and insurance
- [x] #9182 (PR ready)
- [ ] #9196 (Suzanne for writing, CMS engineer, FE engineer)
- [ ] Editorial interface about duplicate labels (probably should ask 508 folks for input on "Billing Office" "Billing Office")

CMS 
- [x] #8337 
- [x] #8648 

## Events V2 and Recurring events

Issues in epic https://github.com/department-of-veterans-affairs/va.gov-team/issues/28918

### #7624 Support lighthouse with unique ID for health service terms / #5017  Push facility services to lighthouse

<details><summary>Blocked by Lighthouse v1 API</summary>

- [ ] #8019 done but not merged, blocked by Lighthouse 
- [ ] #8017  blocked by Lighthouse AND
    - [ ] Field Office Hours hardening https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6931
      - [x] script the migration https://github.com/department-of-veterans-affairs/va.gov-cms/pull/8533
      - [x] run the migration on prod to populate data for FE 
      - [ ] FE builds new display  https://github.com/department-of-veterans-affairs/va.gov-team/issues/39436 
      - [x] After FE goes live(same day) Hide field `field_facility_service_hours` and show 'field_office_hours' in cms  #8550 
      - [x] Re-run script on prod `drush scr scripts/content/VACMS-6931-switch-service-locations-office-hours.php`
      - [ ] Notify editors / process bad entries from the outliers.csv #8708
      - [x] #8709

Possible to add to Lighthouse V0 API

- [ ] Stretch: #4769 

</details>

## For awareness from other teams

- #9097 (Sitewide CMS)

## Refinement

- #9161 (Facility Locator, top priority)
- #9061 (FE defect, Vet Center)
- #9067 (Facility Locator defect)
- #9196 (Drupal & FE tech debt, Billing and insurance)
- #9220 (FE prod defect, Billing and insurance)
- #9207 (Drupal tech debt)
- #9099 (Drupal, implementing UX enhancement)
